frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -100,20 +136,34 @@ async function deleteUserRoleAssignment(
                     },
                 }"
             >
-                <TabPanel header="Users">
+                <TabPanel
+                    header="Application admins"
+                    v-if="selectedApplication?.application_id === FAM_APPLICATION_ID"
```

Comment:
I wonder if here should be "selectedApplicationId"

Reply:
It did not need the .value but I forgot to change that to the selectedApplicationId state